Source/Extensions/Blazorise.Charts.DataLabels/wwwroot/chart.datalabels.js

```diff
@@ -4,23 +4,27 @@ import { getChart } from "../Blazorise.Charts/charts.js?v=1.1.2.0";
 Chart.register(ChartDataLabels);
 
 export function setDataLabels(canvasId, datasets, options) {
-    const chart = getChart(canvasId);
+    Retry(function () { return getChart(canvasId) }, 100, 10).then(chart => {
```

Comment:
Damn... ugly... But I guess it's becaused there's nothing in the DOM yet?
I think in other places we did the await in C# instead. Something like Task.Delay(150) to give enough time to render? But this might be safer I guess.

Reply:
Old commit.